### PR TITLE
Translate more docs in to Bosnian.

### PR DESCRIPTION
**Pull request by NermanLegacy**
Translated the following `Resources` documents:
- anglemodes.md
- boneid.md
- bullethittypes.md
- callbacks-sequence.md
- cameracutstyles.md
- cameramodes.md
- clicksources.md
- Componentslots.md
- spectatetypes.md
- recordtypes.md

Translated the following `Client` document:
- sa-mp.cfg.md

Translated the following `Tutorials` document:
- PortForwarding.md

### CODE REVIEW

**Comment by NermanLegacy** [comment]
@adib-yg @daddyDOT If there are any problems please let me know. Thanks in advance.

File: docs/scripting/functions/SetPlayerMarkerForPlayer.md [hunk 1]
```diff
@@ -40,3 +40,4 @@ SetPlayerMarkerForPlayer(42, 1, (GetPlayerColor(1) | 0x000000FF));
 - [LimitPlayerMarkerRadius](LimitPlayerMarkerRadius): Limit the player marker radius.
 - [SetPlayerColor](SetPlayerColor): Set a player's color.
 - [ShowPlayerNameTagForPlayer](ShowPlayerNameTagForPlayer): Show or hide a nametag for a certain player.
+- [GetPlayerMarkerForPlayer](GetPlayerMarkerForPlayer): Gets a player's marker for a specific player.
```
Comment (adib-yg) [inline docs/scripting/functions/SetPlayerMarkerForPlayer.md:43]:
```suggestion
- [GetPlayerMarkerForPlayer](GetPlayerMarkerForPlayer): Gets the colour of a player's nametag and radar blip for another player.
```

**Review by adib-yg — CHANGES_REQUESTED** [review]

**Review by adib-yg — APPROVED** [review]

File: docs/translations/bs/tutorials/PortForwarding.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+**Dobrodošli u tutorijal za Port Forward-ovanje!**
+
+Dakle, stigli ste do tutoriala za Port Forward-ovanje, koji je napravio Leopard.
+
+Sve što vam treba je samp-server ili omp-server i ruter. Ako nemate ruter, onda ne trebate port forward-ovati!
+
+## Početak
+
+Ok, evo početka, počnite pronalaženjem svog **kapija**. Pretpostavimo da imate vista. Kliknite na start, kliknite na polje za pretragu i napišite **cmd**. Tada se pojavljuje crna kutija. Unesite sljedeće; **ipconfig**. Sačekajte da se tekst učita, a zatim ga pogledajte. Nastavite tražiti dok ne pronađete _**gateway**_, i ne zatvarajte crnu kutiju!
```
Comment (daddyDOT) [inline docs/translations/bs/tutorials/PortForwarding.md:14]: Kapija? 😅
Ostavi 'gateway-a'

**Review by daddyDOT — CHANGES_REQUESTED** [review]
Sve dobro, samo pregledaj i ispravi ovo gdje sam ostavio komentar na "`kapija`".

**Review by NermanLegacy — COMMENTED** [review]
@daddyDOT Ispravio sam gresku.

**Review by daddyDOT — APPROVED** [review]

**Pull request merged**